jest-gae/src/main/java/com/searchly/jestgae/JestGaeClient.java

```diff
@@ -0,0 +1,150 @@
+package com.searchly.jestgae;
+
+import com.google.appengine.api.urlfetch.*;
+import com.google.gson.Gson;
+import io.searchbox.action.Action;
+import io.searchbox.client.AbstractJestClient;
+import io.searchbox.client.JestClient;
+import io.searchbox.client.JestResult;
+import io.searchbox.client.JestResultHandler;
+import org.apache.commons.codec.binary.Base64;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.Map.Entry;
+
+/**
+ * @author Scopewriter
+ */
+public class JestGaeClient extends AbstractJestClient implements JestClient {
+
+    private final static Logger log = LoggerFactory.getLogger(JestGaeClient.class);
+    private final static String DEFAULT_OK_RESPONSE = "{\"ok\" : true, \"found\" : true}";
+    private final static String DEFAULT_NOK_RESPONSE = "{\"ok\" : false, \"found\" : false}";
```

Comment:
Default responses for `HEAD` requests are removed as obsolete, please remove them from gae package too.